### PR TITLE
Merge branch `ACMS-3568` into `develop`.

### PR DESCRIPTION
**Pull request by vishalkhode1**

### CODE REVIEW

**Review by chandan-singh7929 — APPROVED** [review]
Overall all changes looks good to me.

File: tests/src/functional/Drush/Traits/SiteUriTraitTest.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    $this->fileSystem->appendToFile($sites_file, PHP_EOL . '$sites["www.acquia.com/qa/acquia_cms"] = "qa.acquia_cms";');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function tearDown(): void {
+    parent::tearDown();
+    if ($this->content) {
+      file_put_contents($this->getDrupalRoot() . "/sites/sites.php", $this->content);
```
[the remark this answers is not in the document]
Comment (vishalkhode1) [reply tests/src/functional/Drush/Traits/SiteUriTraitTest.php:91]: If we already have `sites.php` file and then if we are running tests, then we are storing the content of sites.php in a variable before making any changes. See [here](https://github.com/acquia/drupal-recommended-settings/pull/37/files/d4658c4097650097409326f8cdcec638d613714e#diff-bd85d7147381c2f06a774d8c409f0903fdd4a070fb89dc3dc25bfcec137e5485R78)
And then our test runs, we are ensuring that the `sites.php` have some file content prior to changes that we've made.

[the remark this answers is not in the document]
Comment (chandan-singh7929) [reply tests/src/functional/Drush/Traits/SiteUriTraitTest.php:91]: As per our discussion please update this section.

**Review by apathak18 — APPROVED** [review]

File: tests/src/functional/Drush/Traits/SiteUriTraitTest.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (vishalkhode1) [reply tests/src/functional/Drush/Traits/SiteUriTraitTest.php]: @chandan-singh7929 Updated the code, please review now.

**Pull request merged**